src/Markup/Avalonia.Markup.Xaml.Loader/AvaloniaRuntimeXamlLoader.cs

```diff
@@ -14,9 +13,9 @@ public static class AvaloniaRuntimeXamlLoader
         /// </summary>
         /// <param name="xaml">The string containing the XAML.</param>
         /// <param name="localAssembly">Default assembly for clr-namespace:</param>
-        /// <param name="rootInstance">
-        /// The optional instance into which the XAML should be loaded.
-        /// </param>
+        /// <param name="rootInstance"> instance into which the XAML should be loaded</param>
```

Comment:
First part of the comment cut off:

```suggestion
        /// <param name="rootInstance">The optional instance into which the XAML should be loaded.</param>
```